packages/split-view/src/SplitView.ts

```diff
@@ -303,12 +303,12 @@ export class SplitView extends SpectrumElement {
         }
     }
 
-    private async checkResize(): Promise<void> {
+    private async checkResize(forceUpdate = false): Promise<void> {
         if (!this.enoughChildren) {
             return;
         }
         this.updateMinMax();
-        if (this.splitterPos === undefined) {
+        if (this.splitterPos === undefined || forceUpdate) {
```

Comment:
I'm not really sure about this, I feel like there could be some better way instead of adding a new boolean argument. Removing this condition breaks the `keeps the splitter pos when removing and re-adding a panel` test.